### PR TITLE
WIP Add Manual link in ShareMenu

### PR DESCRIPTION
**Pull request by edellucien**
# How to

The quickstart Link is supposed to be displayed only for intern topic. But for you to test i display it for public topic first.
Then when you accept the MR I will switch the display for internal and edit the tests.

# Others

- [x] It's not a hack or at least an unauthorized hack :).
- [x] The images added are optimized.
- [x] Everything in ticket description has been fixed.
- [x] The author of the MR has made its own review before assigning the reviewer.
- [x] IE11 tested.
- [x] Labels applied. if it's a release? a hotfix?
- [x] Tests added.

### CODE REVIEW

**Comment by edellucien** [comment]
Previous MR: https://github.com/geops/trafimage-maps/pull/627
Closed to change base to release.1.0.8
Some comments still opened maybe there

File: src/components/Share/Share.js [hunk 1]
```diff
@@ -48,26 +54,36 @@ const renderConf = (conf, t) => (
 );
 
 const Share = ({ appBaseUrl }) => {
+  const activeTopic = useSelector(state => state.app.activeTopic);
+  const language = useSelector(state => state.app.language);
   const map = useSelector(state => state.app.map);
   const { t } = useTranslation();
-  const config = [...socialShareConfig];
+  const duplicateConfig = JSON.parse(JSON.stringify(socialShareConfig));
```
Comment (mhaertwig) [inline src/components/Share/Share.js:61]: As @oterral already wrote: Please don't duplicate the config using `JSON.parse` and `JSON.stringify`.
There are other possibilities to replace the placeholders instantly, e.g. by using a helper function.

`<a href={replaceParams(conf.url)} />` etc.

Comment (edellucien) [reply src/components/Share/Share.js]: Ok I replaced with a helperFunction

File: src/components/Share/__snapshots__/Share.test.js.snap [hunk 2]
```diff
@@ -173,6 +173,37 @@ exports[`Share should match snapshot. 1`] = `
       </svg>
     </a>
   </div>
+  <div
+    className="ta-manual-icon"
+  >
+    <a
+      href="null/static/app_trafimage/docs/undefined/Quickstart.pdf"
```
Comment (oterral) [inline src/components/Share/__snapshots__/Share.test.js.snap:180]: can you make a real url appears? wihtout null and undefined values

**Comment by oterral** [comment]
MInor tets changes

File: src/components/Share/__snapshots__/Share.test.js.snap (hunk 2 above)
Comment (edellucien) [reply src/components/Share/__snapshots__/Share.test.js.snap]: Makes sense, I pass the correct props to the snapshot test.

**Pull request merged**